openapi/components/schemas/Applications/ApplicationInstance.yaml

```
@@ -12,16 +12,21 @@ properties:
       - enabled
       - disabling
       - disabled
-  settings:
+  isConfigured:
     description: |-
-      Settings that are available for the user to complete when they install the application.
-      Use this object to describe fields that are rendered during installation, in the settings form.
+      Determines if the application instance configuration complete.
```

```suggestion
      Determines if the application instance configuration is complete.
```
Or, where possible, avoid using a series of nouns like this, it can be difficult to read
```suggestion
      Determines if the configuration of the application instance is complete.
```